pontoon/sync/vcs/models.py

```diff
@@ -151,36 +151,22 @@ def find_changed_files(repo, locale=None):
                     for path in find_changed_files(repo, locale):
                         files.setdefault(path, []).append(locale)
             else:
+                locale_directories = self.locale_directories(repo)
+                locale_directories_paths = locale_directories.keys()
+
                 for changed_file in find_changed_files(repo):
-                    path_info = self.get_path_info(changed_file, repo)
+                    if is_hidden(changed_file):
```

Comment:
Hm, do you have any exaples of repos that have a large number of hidden files in their structure ?